bin/docker/kafka_bridge_config_generator.sh

```diff
@@ -66,6 +66,10 @@ if [ -n "$KAFKA_BRIDGE_SASL_MECHANISM" ]; then
             OAUTH_CLIENT_SECRET="oauth.client.secret=\"$KAFKA_BRIDGE_OAUTH_CLIENT_SECRET\""
         fi
 
+        if [ ! -z "$KAFKA_BRIDGE_OAUTH_PASSWORD_GRANT_PASSWORD" ]; then
+            OAUTH_PASSWORD_GRANT_PASSWORD="oauth.password.grant.password=\"$KAFKA_BRIDGE_OAUTH_PASSWORD_GRANT_PASSWORD\""
+        fi
+
```

Comment:
You also need to add `${OAUTH_PASSWORD_GRANT_PASSWORD}` here: https://github.com/strimzi/strimzi-kafka-bridge/blob/main/bin/docker/kafka_bridge_config_generator.sh#L73

Reply:
Thanks @mstruk ! I added it, can you review again please?